### ISSUE TITLE
Field setting to manage the "Week Start Day" in the CP

### ISSUE THREAD

**Issue by lindseydiloreto** [craftcms/store-hours#43]
This little plugin has been pretty helpful, thanks! 🙂

Within the control panel, it would be great if we could set the **leading day of the week** for each Store Hours field. This field is currently locked to begin with Sunday, with no clear ability to change it...

<img width="594" alt="Hours in the control panel" title="Hours in the control panel" src="https://user-images.githubusercontent.com/5309692/122282936-d9da8c80-cea0-11eb-83d0-2e23fb39c59b.png">

It leads to a bit of confusion, because on the front-end we are using **Monday** as the first day of the week...

<img width="459" alt="Hours on the front end" title="Hours on the front end" src="https://user-images.githubusercontent.com/5309692/122282509-5620a000-cea0-11eb-9eb0-e603d8908fc0.png">

So what we're seeing on the front-end (or in Live Preview mode) doesn't properly align with how it appears in the CP. This increases the odds that a careless content editor might enter their hours into the wrong subfields.

---

**TL;DR - Would it be possible to add a field setting to manage the "Week Start Day" in the control panel?**

**Comment by brandonkelly** [comment]
Just tagged 4.1.0 with a new “Start Day” field setting.

**Comment by lindseydiloreto** [comment]
Very cool, thanks @brandonkelly! 👍
